<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorAepWidget</class>
 <widget class="QWidget" name="GeneratorAepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>355</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>355</width>
    <height>470</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Генератор АЭП</string>
  </property>
  <widget class="QLineEdit" name="m_numSets_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_size_Complect">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество комплектов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_timeSet_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPreview">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Предварительное время</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Time_Complect">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Перерыв  между комплектами</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_status_l">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Статус:</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="m_loadSets_tw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>190</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">Каталог</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="m_previewTime_le">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="m_editTime_dte">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_clear_pb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_start_pb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>СТАРТ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_saveSets_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_report_pb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчёт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ampl_res">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Размах:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAmplituda">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_amplituda_dsb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_back_pb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../path.qrc">
     <normaloff>:/image/button_back.png</normaloff>:/image/button_back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelPreliminarilyTime">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранение</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../path.qrc"/>
 </resources>
 <connections/>
</ui>
